ethereum/spec/src/main/java/tech/pegasys/teku/spec/constants/ValidatorConstants.java

```diff
@@ -16,7 +16,7 @@
 public class ValidatorConstants {
   // Phase0
   public static final int TARGET_AGGREGATORS_PER_COMMITTEE = 16;
-  public static final int RANDOM_SUBNETS_PER_VALIDATOR = 1;
+  public static final int NODE_ID_BITS = 256;
```

Comment:
It shouldn't be there, the major change in this revamp is that not only validators will subscribe, any node

Reply:
I'll put it `NetworkConstants` then

Reply:
Done